alchemist/alchemist-swingui/src/main/java/it/unibo/alchemist/boundary/monitors/Generic2DDisplay.java

```diff
@@ -309,26 +345,26 @@ protected final void drawEnvOnView(final Graphics2D g) {
              * TODO: only draw obstacles if on view
              */
             obstacles.parallelStream()
-                .map(this::convertObstacle)
-                .forEachOrdered(g::fill);
+                    .map(this::convertObstacle)
+                    .forEachOrdered(g::fill);
         }
         if (paintLinks) {
             g.setColor(Color.GRAY);
             onView.keySet().parallelStream()
-                .map(neighbors::get)
-                .flatMap(neigh -> neigh.getNeighbors().parallelStream()
-                    .map(node -> node.compareTo(neigh.getCenter()) > 0
-                                ? new Pair<>(neigh.getCenter(), node)
-                                : new Pair<>(node, neigh.getCenter())))
-                .distinct()
-                .map(pair -> mapPair(pair, node ->
-                        Optional.ofNullable(onView.get(node))
-                        .orElse(wormhole.getViewPoint(positions.get(node)))))
-                .forEachOrdered(line -> {
-                    final Point p1 = line.getFirst();
-                    final Point p2 = line.getSecond();
-                    g.drawLine(p1.x, p1.y, p2.x, p2.y);
-                });
+                    .map(neighbors::get)
+                    .flatMap(neigh -> neigh.getNeighbors().parallelStream()
+                            .map(node -> node.compareTo(neigh.getCenter()) > 0
+                                    ? new Pair<>(neigh.getCenter(), node)
+                                    : new Pair<>(node, neigh.getCenter())))
+                    .distinct()
+                    .map(pair -> mapPair(pair, node ->
+                            Optional.ofNullable(onView.get(node))
+                                    .orElse(wormhole.getViewPoint(positions.get(node)))))
+                    .forEachOrdered(line -> {
+                        final Point p1 = line.getFirst();
+                        final Point p2 = line.getSecond();
+                        g.drawLine(p1.x, p1.y, p2.x, p2.y);
+                    });
```

Comment:
I do not understand why indenting so much

Reply:
IDEA recognizes this indentation level as correct, I still reverted it.